include/circt/Dialect/MSFT/MSFTPasses.td

```diff
@@ -23,7 +23,7 @@ def ExportTcl: Pass<"msft-export-tcl", "mlir::ModuleOp"> {
   let options = [
     ListOption<"tops", "tops", "std::string",
                "List of top modules to export Tcl for",
-	       "llvm::cl::ZeroOrMore, llvm::cl::MiscFlags::CommaSeparated">,
+	             "llvm::cl::ZeroOrMore,">,
```

Comment:
nit: unrelated format change.

also: do you have a .td formatter you are using? i've wished for such a thing, so the community can standardize on tablegen format like how clang-format makes the c++ format unambiguous.

Reply:
I don't have a formatter, I just inferred that the original indentation was due to the longer line length. Also I didn't realize there was a tab in there, I've updated it to be consistent with the other lines, but I can change it to the original indentation if you prefer.

Looks like a td formatter is [being proposed](https://discourse.llvm.org/t/tablegen-formatter/60418).

Reply:
Gotcha, removing the tab looks good to me.